domain/src/main/assets/about_oppia.json

```diff
@@ -0,0 +1,583 @@
+              "I want to go back to my earlier questions."
+            ]
+          }
+        },
+        "default_outcome": {
+          "dest": "Site License",
+          "feedback": [
+            "..."
+          ],
+          "param_changes": []
+        },
+        "hints": [],
+        "id": "MultipleChoiceInput",
+        "solution": null
+      },
+      "param_changes": []
+    },
+    "So what can I tell you": {
+      "classifier_model_id": null,
+      "content": {
+        "audio_translations": {},
+        "html": "So, what else can I tell you about myself?"
+      },
+      "interaction": {
+        "answer_groups": [
+          {
+            "correct": false,
+            "outcome": {
+              "dest": "About this website",
+              "feedback": [],
+              "param_changes": []
+            },
+            "rule_specs": [
+              {
+                "inputs": {
+                  "x": 1
+                },
+                "rule_type": "Equals"
+              }
+            ]
+          },
+          {
+            "correct": false,
+            "outcome": {
+              "dest": "End Card",
+              "feedback": [],
+              "param_changes": []
+            },
+            "rule_specs": [
+              {
+                "inputs": {
+                  "x": 3
+                },
+                "rule_type": "Equals"
+              }
+            ]
+          },
+          {
+            "correct": false,
+            "outcome": {
+              "dest": "Example1",
+              "feedback": [],
+              "param_changes": []
+            },
+            "rule_specs": [
+              {
+                "inputs": {
+                  "x": 0
+                },
+                "rule_type": "Equals"
+              }
+            ]
+          },
+          {
+            "correct": false,
+            "outcome": {
+              "dest": "Contribute",
+              "feedback": [],
+              "param_changes": []
+            },
+            "rule_specs": [
+              {
+                "inputs": {
+                  "x": 2
+                },
+                "rule_type": "Equals"
+              }
+            ]
+          }
+        ],
+        "confirmed_unclassified_answers": [],
+        "customization_args": {
+          "choices": {
+            "value": [
+              "How do your explorations work?",
+              "What can you tell me about this website?",
+              "How can I contribute to Oppia?",
+              "Those were all the questions I had!"
+            ]
+          }
+        },
+        "default_outcome": {
+          "dest": "So what can I tell you",
+          "feedback": [
+            "..."
+          ],
+          "param_changes": []
+        },
+        "hints": [],
+        "id": "MultipleChoiceInput",
+        "solution": null
+      },
+      "param_changes": []
+    }
+  },
+  "states_schema_version": 13,
```

Comment:
Since we don't have a migration pipeline yet, should we upgrade these to their latest version? Per https://github.com/oppia/oppia/blob/develop/feconf.py#L179 I believe the current version is 30, and that's what we've defined our proto schema against.

Reply:
Ah you are right. 
I dont know how to get the fields that are missing such as written translations / recorded voiceovers etc. What I have done is converted the current json file into the version 30 schema.
I will look for newer explorations as well that contain audio and translations